AdminWebsite/AdminWebsite/Controllers/AudioPlatformController.cs

```diff
@@ -50,14 +50,7 @@ public async Task<IActionResult> GetAudioRecordingLinkAsync(Guid hearingId)
             try
             {
                 var requestKey = "";
-                if (_featureToggles.HrsEnabled())
-                {
-                    requestKey = await GetAudioHrsFileName(hearingId);
-                }
-                else
-                {
-                    requestKey = hearingId.ToString();
-                }
+                requestKey = await GetAudioHrsFileName(hearingId);
```

Comment:
Please remove `IFeatureToggles` from the constructor and in the test setup

Reply:
done